src/Style/Input.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const Input = styled.input`
+  padding: 0.5em;
+  margin: 0.5em;
+  color: "palevioletred";
+  background: papayawhip;
```

Comment:
Here one color has double quotes but the other doesn't. It's not necessary to use them